www/src/layouts/index.js

```diff
@@ -132,14 +135,52 @@ class DefaultLayout extends React.Component {
           />
           <html lang="en" />
         </Helmet>
+        <div
+          css={{
+            width: `100%`,
+            padding: rhythm(1 / 2),
+            background: isHomepage ? `#402060` : colors.gatsby,
+            color: colors.ui.bright,
+            fontFamily: options.headerFontFamily.join(`,`),
+            fontSize: scale(-1 / 5).fontSize,
+            zIndex: `3`,
+            [presets.Desktop]: {
+              position: `fixed`,
+            },
+            [presets.Tablet]: {
+              position: `fixed`,
+            },
+            WebkitFontSmoothing: `antialiased`,
+          }}
+        >
+          These are the docs for V1.{` `}
+          <OutboundLink
+            href="https://nextgatsbyjs.org/"
+            css={{
+              color: `#fff`,
+            }}
+          >
+            View the v2 docs
+            <span
+              css={{
+                display: `none`,
+                [presets.Mobile]: {
+                  display: `inline`,
+                },
+              }}
+            >
+              {` `}
+              instead
+            </span>
+          </OutboundLink>.
```

Comment:
Same changes to this section.

Also: I'm not sure if this is how the v2 banner is set up, but why is this in two places? @m-allanson @fk @pieh?

Reply:
@jlengstorf  I missed to delete a file... I will delete it